pages/about.tsx

```diff
@@ -34,7 +35,9 @@ const About: NextPage<Props> = () => {
         <link rel="icon" href="/dark-logo.svg" />
       </Head>
       <Navbar active={'about'} />
-      <main tw="flex mt-[8rem]  lg:w-full flex-1 "></main>
+      <main tw="flex mt-[8rem]  lg:w-full flex-1 ">
+        <Toaster />
```

Comment:
could you check if we can remove this? see without this if our toast message is showing or not